src/api/java/com/minecolonies/api/util/LanguageHandler.java

```diff
@@ -135,7 +135,7 @@ public static String format(final String key, final Object... args)
      * @param key     key of the message.
      * @param message the message.
      */
-    public static void sendPlayersMessage(@Nullable final List<EntityPlayer> players, final String key, final Object... message)
+    public static void sendPlayersMessage(@Nullable final Set<EntityPlayer> players, final String key, final Object... message)

```

Comment:
Again:
  * How about global subscribers or new subscribers?

Reply:
playersMessage should be more general, it could be related to anything, not just subscriber messages